src/app/pages/template/stripe-template/stripe-template.component.ts

```diff
@@ -0,0 +1,168 @@
+  templateUrl: './stripe-template.component.html',
+  styleUrls: ['./stripe-template.component.scss']
+})
+
+export class StripeTemplateComponent implements OnInit {
+
+  // Billing & Billing Information
+  public isActive = false;
+  public billingSettings!: BillingSettings;
+
+  @ViewChild('cardInfo', { static: true }) cardInfo: ElementRef;
+
+  feedback: any;
+  stripe: Stripe;
+  elements: StripeElements;
+  card: StripeCardElement;
+
+  //constructor(private stripeScriptService: StripeScriptService) {}
+  constructor(
+    private centralServerService: CentralServerService,
+    private componentService: ComponentService,
+    // private dialogService: DialogService,
+    private messageService: MessageService,
+    private spinnerService: SpinnerService,
+    // private translateService: TranslateService,
+    private router: Router,
+  ) {
+    this.isActive = this.componentService.isActive(TenantComponents.BILLING);
+  }
+
+  ngOnInit(): void {
+    this.initialize();
+  }
+
+  async initialize(): Promise<void> {
+
+    try {
+
+      this.spinnerService.show();
+      this.billingSettings = await this.loadBillingConfiguration();
+      if ( !this.billingSettings || !this.billingSettings.stripe ) {
+        this.feedback = "Stripe configuration is not set for the current tenant";
+      } else {
+        this.stripe = await this.initializeStripe(this.billingSettings.stripe.publicKey);
+        this.initializeCardElements();
+      }
+    }
+    catch (error) {
+      Utils.handleHttpError(error, this.router, this.messageService, this.centralServerService, 'general.unexpected_error_backend');
+    }
+    finally {
+      this.spinnerService.hide();
+    }
+
+  }
+
+  async initializeStripe(publicKey: string) : Promise<Stripe> {
+
+    loadStripe.setLoadParameters({ advancedFraudSignals: false })
+    return await loadStripe(publicKey);
+  }
+
+  initializeCardElements() {
+
+    this.elements = this.stripe.elements();
+    this.card = this.elements.create('card');
+    this.card.mount(this.cardInfo.nativeElement);
+  }
+
+  async loadBillingConfiguration(): Promise<BillingSettings> {
+
+    try {
+
+      return await this.componentService.getBillingSettings().toPromise();
+
+    } catch (error) {
+
+      switch (error.status) {
+        case HTTPError.OBJECT_DOES_NOT_EXIST_ERROR:
+          this.messageService.showErrorMessage('settings.billing.not_found');
+          break;
+        default:
+          Utils.handleHttpError(error, this.router, this.messageService,
+            this.centralServerService, 'general.unexpected_error_backend');
+      }
+    }
+  }
+
+  linkCardToAccount() {
+    this.doCreatePaymentMethod();
+  }
+
+  async doCreatePaymentMethod() {
+
+    const operationResult: any = await this.createPaymentMethod()
+    if (operationResult.error) {
+      // Operation failed
+      this.feedback = operationResult.error.message;
+    } else {
+      // Operation succeeded
+      this.feedback = operationResult;
+
```

Comment:
Can be also done this way:
this.feedback = operationResult.error ? operationResult.error.message : operationResult;